pkgs/by-name/ya/yatto/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule rec {
```

Lets use the finalAttrs pattern which enable better override flexibility
```suggestion
buildGoModule (finalAttrs: {
```